website/docs/r/network_routed.html.markdown

```diff
@@ -13,6 +13,10 @@ modify, and delete internal networks for vApps to connect.
 
 Supported in provider *v2.0+*
 
+-> **Note:** This resource supports only NSX-V backed Org VDC networks and is **deprecated**.
```

Comment:
What about using more eye-poking yellow box for these deprecations? Hopefully that would inspire more users to switch to the new resource.

Reply:
Yes. 
<img width="979" alt="image" src="https://user-images.githubusercontent.com/15804230/106649889-de11a780-659a-11eb-9917-f02b4deff1f3.png">
<img width="971" alt="image" src="https://user-images.githubusercontent.com/15804230/106649949-f1bd0e00-659a-11eb-8647-f72ea9fc47fd.png">